### DISCUSSION TITLE
Question on Monitoring of individual devices - Emporia View & other monitors

### DISCUSSION THREAD

**Discussion by Infernoken76** [emporia-vue-local/esphome#161] [General]
Hi,

I'm further adapting my installation and have now fully installed the Emporia View & loving it !!!

One question I have for the Energy View - Monitoring of individual devices is how I can substract the other power monitors (power plugs with monitoring function) I have on certain circuits from the circuits that the Emporia view is showing?

Otherwise it would overreport the total usage of individual circuits/devices as they are double counted...

Any idea ???

**Comment by flaviut** [comment]
I would recommend simply not including the individual device monitors in the Home Assistant Energy dashboard.

You can still use them for automations and historical data.

The other way to do this would be to use [the template integration in Home Assistant](https://www.home-assistant.io/integrations/template/), but that's not something I can help with.
